Portal/src/Datahub.Portal/Layout/Topbar/Topbar.razor

```diff
@@ -32,7 +32,7 @@
                 </li>
                 <li style="display: inline-block; margin-inline: 5px">
                     <DHButton Variant="Variant.Text" Href=@Localizer[PageRoutes.News] Size="Size.Small">
-                        @Localizer["News"]
+                        @Localizer["Resources"]
```

Comment:
I might be missing some context but why is news called resources now?

Reply:
Good catch, I swapped the wrong label